docs/vsphere_ci/scripts/install-openssh.ps1

```diff
@@ -0,0 +1,56 @@
+# Powershell script to configure OpenSSH Server in Windows Server 1809 and later.
+#
+# USAGE
+#    ./install-openssh.ps1 
+#    ./install-openssh.ps1 <path/to/authorized_keys>
+#    ./install-openssh.ps1 -keyfile=<path/to/authorized_keys>
+#
+# OPTIONS
+#    keyfile        Path to `authorized_keys` file (Default: authorized_keys)
+#                   The `authorized_keys` file must contains the public key corresponding to 
+#                   the private key in the `cloud-private-key` secret on the OpenShift cluster.
+#                   See https://github.com/openshift/windows-machine-config-operator#usage
+
+# define param for authorized_keys file path
+param ($keyfile='authorized_keys')
+# validate given keyfile
+if (-not(Test-Path -Path $keyfile -PathType Leaf)) {
+    # log error and stop
+    Write-Error -ErrorAction Stop -Message "Cannot find authorized key file: $keyfile" 
+}
+
+# install OpenSSH server (See: https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+# set service startup type
+Set-Service -Name ssh-agent -StartupType 'Automatic'
+Set-Service -Name sshd -StartupType 'Automatic'
+# start service
+Start-Service ssh-agent
+Start-Service sshd
+# configure key based-authentication
+$sshdConfigFilePath = "$env:ProgramData\ssh\sshd_config"
+$pubKeyConf = (Get-Content -path $sshdConfigFilePath) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
+$pubKeyConf | Set-Content -Path $sshdConfigFilePath
+$passwordConf = (Get-Content -path $sshdConfigFilePath) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
+$passwordConf | Set-Content -Path $sshdConfigFilePath
+# create authorized_keys file in configuration
+$authorizedKeyConf = "$env:ProgramData\ssh\administrators_authorized_keys"
+New-Item -Force $authorizedKeyConf
+# setup the provided authorized public key
+Get-Content $keyfile | Out-File $authorizedKeyConf -Encoding ascii
+# configure authorized_keys file acl
+$acl = Get-Acl $authorizedKeyConf
+# disable inheritance
+$acl.SetAccessRuleProtection($true, $false)
+# set full control for Administrators
+$administratorsRule = New-Object system.security.accesscontrol.filesystemaccessrule("Administrators","FullControl","Allow")
+$acl.SetAccessRule($administratorsRule)
+# set full control for SYSTEM
+$systemRule = New-Object system.security.accesscontrol.filesystemaccessrule("SYSTEM","FullControl","Allow")
+$acl.SetAccessRule($systemRule)
+# apply file acl
+$acl | Set-Acl
+# restart service
+Restart-Service sshd
+# success
```

Comment:
RE: https://github.com/openshift/windows-machine-config-operator/pull/466/files#r656474446

Should you put a check here to see if the FW rule was created?

Reply:
@christianh814  thanks for the feedback. As per [discussion](https://github.com/openshift/windows-machine-config-operator/pull/466#discussion_r656479631) and the fact that needs more testing/debugging I'm leaning towards to add the validation as a supporting code snippet in the documentation.

Reply:
So it is possible that in the future we will prepare an image and this rule won't be present. Not sure if we want to do that especially given the Microsoft [docs](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse#start-and-configure-openssh-server) also seem to imply that this rule creation is flaky.

Reply:
Agree. 

Added both [a link to Microsoft docs](https://github.com/openshift/windows-machine-config-operator/blame/8f34eca7268bcba6241f52efbee08aae06ef5272/docs/vsphere-golden-image.md#L69) and the validation/creation [code snippets](https://github.com/openshift/windows-machine-config-operator/blame/8f34eca7268bcba6241f52efbee08aae06ef5272/docs/vsphere-golden-image.md#L94) to make it explicit in the docs.

Reply:
My point was we might want to add the check here given the docs is asking the user to check for the rule. But I will let this go. We just need to be aware that we might be creating flaky images in the future.